vllm/engine/arg_utils.py

```diff
@@ -136,28 +139,34 @@ def is_not_builtin(type_hint: TypeHint) -> bool:
 
 def get_kwargs(cls: ConfigType) -> dict[str, Any]:
     cls_docs = get_attr_docs(cls)
+    resolved = get_type_hints(
+        cls,
+        globalns=sys.modules[cls.__module__].__dict__,
+        include_extras=True,
+    )
```

Comment:
Is this necessary because you set the types to `Any` in `config.py`?

Reply:
yes